### PR TITLE
Issd 1822 2

### PR DESCRIPTION
**Pull request by amosfong**
I made a bunch of SF unrelated to this PR too

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/navigation-primary/ddm-template.ftl [hunk 1]
```diff
@@ -4,11 +4,11 @@
 			<#if (navPrimaryItem.getChildren()?size > 0)>
 				<div class="adt-nav-item dropdown dropdown-action w-100">
 					<button
-							class="adt-nav-text align-items-center d-flex menu-info"
-							data-toggle="liferay-dropdown"
-							id="main-menu-id"
-							aria-expanded="true"
-							tabindex="4"
+						class="adt-nav-text align-items-center d-flex menu-info"
```
Comment (ryanschuhler) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/navigation-primary/ddm-template.ftl:7]: @samuelleolopes, we should sort these attributes alphabetically

**Comment by ryanschuhler** [comment]
formatting wise and looking through this pull looks good. There seems to be a lot of changes, but nothing raises concerns for me at this point. I dont have insight into the architecture yet, so was able to test anything, but from a base level review this looks good to me

**Comment by amosfong** [comment]
@ryanschuhler added new commit with changes

**Comment by ryanschuhler** [comment]
okay looks good from what I can tell, @amosfong do you want me to push it through to Brian? or did you have anything else you were still looking into?

**Comment by amosfong** [comment]
cool, just checking if you had anything else. I can forward it from here

**Comment by amosfong** [comment]
ci:forward:force

**Pull request closed without merging**